src/engine/engine.h

```diff
@@ -154,7 +154,7 @@ namespace CityFlow {
 
         void setTrafficLightPhase(const std::string &id, int phaseIndex);
 
-        void reset();
+        void reset(bool resetRnd = false);
```

Comment:
Is it necessary to use the same parameter name in .h and .cpp files?  (resetRnd and resetRand)